### ISSUE TITLE
Remember sorting

### ISSUE THREAD

**Issue by idleberg** [tomahawk-player/tomahawk#585]
I usually use Tomahawk in song view with my music sorted by „Age”. Unfortunately, I have to configure my preferred sorting every time I launch Tomahawk. It would be great if Tomahawk would remember the last used sorting (and perhaps the view as well)

**Comment by muesli** [comment]
It should store it per view already - are we talking about the collection view?

**Comment by idleberg** [comment]
Yes, I'm talking about the Collection view. Whenever I launch Tomahawk, I get to see the Feed view. When I switch to Collection View, the last set sorting isn't remembered. I suspected that missing files might have caused this (part of my music is on a NAS that doesn't run 24/7), but that's not the cause.

macOS 10.12.2
Tomahawk 08.4
